onnxruntime/contrib_ops/cuda/custom_layer_norm.cu

```diff
@@ -0,0 +1,322 @@
+/**
+* Copyright (c) 2016-present, Facebook, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This kernel is from deepspeed, right? 
Why FB and Nvidia header here?

Reply:
Removed FB and nvidia headers